cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/bulkwriter/CassandraBulkWriterContext.java

```diff
@@ -135,12 +135,11 @@ private void publishInitialJobStats(String sparkVersion)
     {
         Map<String, String> initialJobStats = new HashMap<String, String>() // type declaration required to compile with java8
         {{
-            put("jobId", jobInfo.getId().toString());
             put("sparkVersion", sparkVersion);
-            put("keyspace", jobInfo.getId().toString());
-            put("table", jobInfo.getId().toString());
+            put("keyspace", jobInfo.getId());
```

Comment:
is the `jobInfo.getId()` the keyspace? shouldn't we use `qualifiedTableName().keyspace()` here instead?

Reply:
Yes, probably got mixed-up during rebase. Corrected.